schemas/sdk_metadata.sql

```diff
@@ -63,7 +63,8 @@ INSERT INTO sdk_type_info (type, description) VALUES
                                               ('client-side', 'Primarily used for user-facing application.'),
                                               ('server-side', 'Primarily used for server-side applications.'),
                                               ('edge', 'Primarily used to delivery flag payloads to edge services.'),
-                                              ('relay', 'Special case for Relay Proxy.');
+                                              ('relay', 'Special case for Relay Proxy.'),
+                                              ('open-feature-provider', 'Used for OpenFeature providers.');
```

Comment:
Just doing one category for now. It isn't clear how this will evolve. It is possible that we could make a provider that worked for both paradigms.